src/main/java/com/revature/services/PersonLogicImpl.java

```diff
@@ -80,8 +80,10 @@ public Person updatePerson(Person p) {
 
 
 	@Override
-	public void deletePerson(Person p) {
-		dao.delete(p.getId());
+	public String deletePerson(int id) {
+		Person pers = getPersonById(id);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "pers" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "pers". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1854)